src/shogun/labels/MulticlassLabels.cpp

```diff
@@ -25,6 +25,14 @@ CMulticlassLabels::CMulticlassLabels(CFile* loader) : CDenseLabels(loader)
 	init();
 }
 
+CMulticlassLabels::CMulticlassLabels(CBinaryLabels* labels) : CDenseLabels(labels->get_num_labels())
+{
+	init();
+
+	for (index_t i = 0; i < labels->get_num_labels(); ++i)
```

Comment:
nice!
@MikeLing this is what I had in mind as one of the things in label refactoring